pkg/filemap/filemap.go

```diff
@@ -0,0 +1,31 @@
+// filemap Defines the Filemap type and its methods.
+package filemap
+
+// File represents a file which was referenced in the issue to be updated.
+type File struct {
+	Path    string `json:"path"`
+	Content string `json:"content"`
+	Tag     string `json:"tag"`
+}
+
+// Filemap represents a mapping of files in a directory by their tagnames.
+type Filemap struct {
```

Comment:
WDYT about renaming to Fileset? I thought about it some more and I think from use perspective this is just a set of files, not really a map.

Reply:
It can still be implemented as a golang map, since there's no generic set type in golang

Reply:
I don't think that makes much sense in the context of this data structure because we're having to map the filetags extracted from the issue to the files themselves.